examples/http_server/module/src/lib.rs

```diff
@@ -17,25 +17,39 @@
 //! Simple example starting an Oak Application serving a static page over HTTP.
 
 use log::{info, warn};
-use oak::{http::Invocation, io::ReceiverExt, CommandHandler, OakError, OakStatus};
+use oak::{http::Invocation, CommandHandler, Label, OakError, OakStatus};
 use oak_abi::proto::oak::application::ConfigMap;
 
-oak::entrypoint!(oak_main<ConfigMap> => |_receiver| {
-    oak::logger::init_default();
-    let node = StaticHttpServer;
-    info!("Starting HTTP server pseudo-node on port 8080.");
-    let http_channel =
-        oak::http::init("[::]:8080").expect("Could not create HTTP server pseudo-Node!");
+oak::entrypoint_command_handler!(oak_main => Main);
 
-    oak::run_command_loop(node, http_channel.iter());
-});
+#[derive(Default)]
+struct Main;
 
-/// A simple HTTP server that responds with `OK` (200) to every request sent to `/`, and with
+impl oak::CommandHandler for Main {
+    type Command = ConfigMap;
+
+    fn handle_command(&mut self, _command: ConfigMap) -> anyhow::Result<()> {
+        let http_handler_sender = oak::io::entrypoint_node_create::<StaticHttpHandler>(
+            "router",
```

Comment:
We don't yet have a router node in this example. I think `handler` is a better name for this node.

Reply:
Good catch, it was my mistake copy-pasting.